digest/layout.py

```diff
@@ -247,6 +254,63 @@ def status_legend_card():
     )
 
 
+def filtering_syntax_help_collapse():
+    """Generates the collapse element that displays syntax help for built-in datatable filtering."""
+    return html.Div(
+        [
+            dbc.Button(
+                [
+                    html.I(
+                        id="filtering-syntax-help-icon",
+                        className="bi bi-caret-right-fill me-1",
+                    ),
+                    "Built-in datatable filtering syntax",
+                ],
+                color="link",
+                id="filtering-syntax-help-button",
+                n_clicks=0,
+                className="ps-0",
+            ),
+            dbc.Collapse(
+                dbc.Card(
+                    html.P(
+                        [
+                            dcc.Markdown(
+                                "To filter column values in the table below, supported operators include: `contains` (default), `=`, `>`, `<`, `>=`, `<=`, `!=`. To filter a column for missing (empty) values, use `is blank`.",
```

Comment:
```suggestion
                                "To filter column values in the table below, "
                                "supported operators include: `contains` (default),"
                                " `=`, `>`, `<`, `>=`, `<=`, `!=`." 
                                "To filter a column for missing (empty) values, use `is blank`." 
                                "There is no filter for `is not blank`. This is a know limitation.",
```
> "There is no filter for `is not blank`. This is a know limitation."

Suggest adding  or something like it. 

Also :cherries: on the line length